src/isIsogram.test.js

```diff
@@ -3,7 +3,35 @@
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
+  it('should be declared', () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for a word with no repeating letters', () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('Adam')).toBe(false);
+  });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('Oops')).toBe(false);
```

Comment:
The test case for 'Oops' also assumes case sensitivity. Ensure that the `isIsogram` function handles different cases correctly, or adjust the input to match the expected behavior.